### PR TITLE
Update to tf 0.13

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
A quick update to make this repo work with the new tf13.

### CODE REVIEW

**Comment by eeeady** [comment]
Whoops. Guess we missed this one. One sec while I make some branches/releases. I'll merge this in and cut a new release after that. Thanks Chris!

**Comment by eeeady** [comment]
Oh actually... if we just change the version strings to be more accepting... it should be fine since this module doesn't really have much to it to break in 13

File: versions.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = "~> 0.13"
```
Comment (eeeady) [inline versions.tf:2]:
```suggestion
  required_version = ">= 0.12.7, < 0.14"
```

File: versions.tf [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
   required_providers {
-    aws = "~> 2.70"
+    aws = "~> 3.0"
```
Comment (eeeady) [inline versions.tf:5]:
```suggestion
    aws = ">= 2.68, < 4.0"
```

**Review by eeeady — CHANGES_REQUESTED** [review]
This should make the module work with 12 and 13... as well as aws provider 2 and 3

**Review by eeeady — APPROVED** [review]
Thank you for the quick turn around!

**Comment by chrisgilmerproj** [comment]
Updated and tested, thanks @eeeady

**Comment by eeeady** [comment]
New release published: v1.2.0

**Pull request merged**